_includes/pbc/all/install-features/202304.0/install-warehouse-picking-feature.md

```diff
@@ -0,0 +1,545 @@
+| PickingListCollectionResponse   | class    | created | src/Generated/Shared/Transfer/PickingListCollectionResponse   |
+| GeneratePickingListsRequest     | class    | created | src/Generated/Shared/Transfer/GeneratePickingListsRequest     |
+| PickingListOrderItemGroup       | class    | created | src/Generated/Shared/Transfer/PickingListOrderItemGroup       |
+| StockCollection                 | class    | created | src/Generated/Shared/Transfer/StockCollection                 |
+| StockConditions                 | class    | created | src/Generated/Shared/Transfer/StockConditions                 |
+| ApiPickingListRequestAttributes | class    | created | src/Generated/Shared/Transfer/ApiPickingListRequestAttributes |
+| ApiPickingListAttributes        | class    | created | src/Generated/Shared/Transfer/ApiPickingListAttributes        |
+| ApiPickingListItemAttributes    | class    | created | src/Generated/Shared/Transfer/ApiPickingListItemAttributes    |
+| ApiPickingListRequestAttributes | class    | created | src/Generated/Shared/Transfer/ApiPickingListRequestAttributes |
+| ApiPickingListRequestAttributes | class    | created | src/Generated/Shared/Transfer/ApiPickingListRequestAttributes |
+| ApiPickingListRequestAttributes | class    | created | src/Generated/Shared/Transfer/ApiPickingListRequestAttributes |
+| Item.uuid                       | property | created | src/Generated/Shared/Transfer/Item                            |
+| User.uuid                       | property | created | src/Generated/Shared/Transfer/User                            |
+| Stock.pickingListStrategy       | property | created | src/Generated/Shared/Transfer/Stock                           |
+
+{% endinfo_block %}
+
+### 3) Set up configuration
+
+1. To make the `picking-lists` and `picking-list-items` resources protected, adjust the protected paths configuration:
+
+**src/Pyz/Shared/GlueBackendApiApplicationAuthorizationConnector/GlueBackendApiApplicationAuthorizationConnectorConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Shared\GlueBackendApiApplicationAuthorizationConnector;
+
+use Spryker\Shared\GlueBackendApiApplicationAuthorizationConnector\GlueBackendApiApplicationAuthorizationConnectorConfig as SprykerGlueBackendApiApplicationAuthorizationConnectorConfig;
+
+class GlueBackendApiApplicationAuthorizationConnectorConfig extends SprykerGlueBackendApiApplicationAuthorizationConnectorConfig
+{
+    /**
+     * @return array<string, mixed>
+     */
+    public function getProtectedPaths(): array
+    {
+        return [
+            '/\/picking-lists(?:\/[^\/]+)?\/?$/' => [
+                'isRegularExpression' => true,
+                'methods' => [
+                    'get',
+                    'getCollection',
+                    'patch',
+                ],
+            ],
+            '/\/picking-lists/[^\/]+/picking-list-items(?:\/[^\/]+)?$' => [
```

Comment:
I'm not clear a bit. Why does the description of these two roles not align with the main integration PR?
https://github.com/spryker/suite-nonsplit/pull/7525

Reply:
Fixed